_tools/parse_version.js

```diff
@@ -0,0 +1,17 @@
+const fs = require("fs/promises");
+
+async function parse() {
+  const content = await fs.readFile("version.go", "utf8");
+  const regex = new RegExp(`^const *version *= *"(?<version>.*)?"`, "m");
+  const version = regex.exec(content)?.groups?.version;
+  if (version === undefined) {
+    process.on("exit", function() {
```

Comment:
[question] why do you use `process.on` instead of just writing process.exist directly?